firestore/src/android/firestore_android.cc

```diff
@@ -206,6 +223,14 @@ enum class InitialLogState {
   kSetDisabled,
 } initial_log_state = InitialLogState::kUnset;
 
+Local<LoadBundleTaskInternal> CreateLoadBundleTask(Env& env,
```

Comment:
Add a test that calls `CreateLoadBundleTask()` with a string that is invalid UTF-8 encoding. In this case, it should return an invalid (null?) object.

Reply:
Done.